src/devol.cpp

```diff
@@ -103,6 +107,18 @@ void devol(double VTR, double f_weight, double f_cross, int i_bs_flag,
             t_tmpP = ta_oldP.col(i);            // t_tmpP is the vector to mutate and eventually select
 
             permute(ia_urn2.memptr(), urn_depth, i_NP, i, ia_urntmp.memptr()); // Pick 4 random and distinct 
+            
+            //Trigger JADE algorithm when d_c <> 0 (randomize cross-over and weight coefficient)
+            if(d_c>0){
+              rand_cross = Rcpp::as<double>(Rcpp::rnorm(1, f_cross, 0.1));
```

Comment:
I should have caught that earlier. I'll just make the change post-merge -- we do have _another_ set of RNGs in Rcpp which are behind the `R::` namespace and which map the (non-vectorized) calls of the C API of R itself better.